### ISSUE TITLE
Missing subtractions on wall (example file)

### ISSUE THREAD

**Issue by johanrd** [IfcOpenShell/IfcOpenShell#618]
2. Convert [618--wall--missing-subtractions.ifc](https://github.com/IfcOpenShell/files/blob/master/618--wall--missing-subtractions.ifc) with 12e9f7b941e0fca8d4f5ee97e5204c2c5c9eb30c (v0.6.0 June 15) and experience that there are no subtractions:
<img width="300" alt="Screenshot 2019-06-17 at 16 21 08" src="https://user-images.githubusercontent.com/4601554/59611719-1b05fd80-911c-11e9-84ad-0ab30fc20f41.png">

```
[Notice] {31pNMFw4X0XRkOF0ThZ3Wd} Boolean operation yields non-manifold result
[Error] {31pNMFw4X0XRkOF0ThZ3Wd} Opening subtraction failed for 15 openings
```

2. Expected geometry from Solibri:
<img width="300" alt="Screenshot 2019-06-17 at 16 23 32" src="https://user-images.githubusercontent.com/4601554/59611792-45f05180-911c-11e9-9475-281e3092307a.png">

(I have yet to find a version that outputs expected geometry, thus the input file is not augmented with `--calculate-quantities`)

**Comment by Moult** [comment]
Checked in latest v0.7.0 and issue still present for the record.

**Comment by aothms** [comment]
Yes it's still a fundamental problem in ifopsh. Non-manifold situations that stem from circular openings aren't really supported atm.

![afbeelding](https://user-images.githubusercontent.com/1096535/142165900-dc9aa01d-0eed-4b16-9551-a392d600c61f.png)

**Comment by johanrd** [comment]
```
./IfcConvert 618--wall--missing-subtractions.ifc 
IfcOpenShell IfcConvert 0.8.4-e8eb5e4 (OCC 7.8.1)
Scanning file...
Done scanning file   
Done resolving references
Parsing input file took 0 seconds
Creating geometry...
Done creating geometry (1 objects)
```

<img width="685" height="548" alt="Image" src="https://github.com/user-attachments/assets/36d3c7e0-b808-4482-88ee-5ef2ba05b435" />
